native/src/components/PrimaryTextButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react'
+import { Button } from 'react-native-elements'
+import { IconNode } from 'react-native-elements/dist/icons/Icon'
+import styled, { useTheme } from 'styled-components/native'
+
+const Container = styled.View<{ $padding: boolean }>`
+  padding: ${props => (props.$padding ? '16px' : '0')};
+`
+
+type PrimaryTextButtonProps = {
+  onPress: () => void
+  text: string
+  disabled?: boolean
+  Icon?: IconNode
+  padding?: boolean
+}
+
+const PrimaryTextButton = ({
+  onPress,
+  text,
+  padding = true,
+  disabled = false,
+  Icon,
+}: PrimaryTextButtonProps): ReactElement => {
+  const theme = useTheme()
+
+  return (
+    <Container $padding={padding}>
```

Comment:
why don't you use the `containerStyle` that the library offers and pass a style instead of padding. That makes the component more flexible. Even you can define a default style.